packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -0,0 +1,490 @@
+      </FtlMsg>
+
+      {/* AccountSuggestion is only shown to Pocket clients */}
+      {isAccountSuggestionBannerVisible && (
+        <div className="info mt-4">
+          <FtlMsg id="signup-info-banner-for-pocket">
+            <p className="info-text">
+              Why do I need to create this account?{' '}
+              <LinkExternal
+                href="https://support.mozilla.org/kb/pocket-firefox-account-migration"
+                className="underline"
+              >
+                Find out here
+              </LinkExternal>
+            </p>
+          </FtlMsg>
+          <FtlMsg id="signup-info-banner-dismiss-button">
+            <button
+              type="button"
+              aria-label="Close"
+              onClick={() => setIsAccountSuggestionBannerVisible(false)}
+            >
+              <IconClose className="fill-black w-3 h-3" role="img" />
+            </button>
+          </FtlMsg>
+        </div>
+      )}
+
+      <div className="mt-4 mb-6">
+        <p className="break-all">{email}</p>
+
+        {canChangeEmail && (
+          <FtlMsg id="signup-change-email-link">
+            <a href="/" className="link-blue text-sm">
+              Change email
+            </a>
+          </FtlMsg>
+        )}
+      </div>
+
+      <FormPasswordWithBalloons
+        {...{
+          formState,
+          errors,
+          trigger,
+          register,
+          getValues,
+          newPasswordErrorText,
+          setNewPasswordErrorText,
+          onFocus,
+          email,
+          newPasswordLabel,
+          confirmPasswordLabel,
+          submitButtonText,
+          onFocusMetricsEvent,
+        }}
+        onSubmit={handleSubmit(onSubmit)}
+        loading={false}
+        showConfirmPwdBalloon
+      >
+        {/* Everything below gets passed to FormPassword as children.
+            We might want to review how this is handled */}
+        {/* TODO: improve a11y - original component had a SR-only label
+        "How old are you? To learn why we ask for your age, follow the “why do we ask” link below. */}
+        <InputText
+          name="userAge"
+          label={ageCheckLabel}
+          inputMode="numeric"
+          className="text-start"
+          pattern="[0-9]*"
+          maxLength={3}
+          onChange={(e) => {
+            // TODO: allow only digits in input field
+            setAge(parseInt(e.target.value));
+            // clear error tooltip if user types in the field
+            if (ageCheckErrorText) {
+              setAgeCheckErrorText('');
+            }
+          }}
+          onBlurCb={onBlurAgeInput}
+          errorText={ageCheckErrorText}
+          anchorStart
+          required
+        />
+        <FtlMsg id="signup-coppa-check-explanation-link">
+          <LinkExternal
+            href="https://www.ftc.gov/business-guidance/resources/childrens-online-privacy-protection-rule-not-just-kids-sites"
+            className="link-blue text-start text-sm mb-4"
```

Comment:
There's a smidge too much space between this and "Practical knowledge", it should be a total of `mb-8`:

![image](https://user-images.githubusercontent.com/13018240/214702336-b85dd803-49aa-4791-a173-669d4fc92f61.png)

vs:

<img width="476" alt="image" src="https://user-images.githubusercontent.com/13018240/214702392-2462757f-7451-458e-8596-07edb8d0a57f.png">